### PR TITLE
Extend spellchecking to the policies

### PR DESCRIPTION
**Pull request by tas50**
Make sure we apply the same ruls to our policies

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: core/mondoo-linux-security.mql.yaml [hunk 1]
```diff
@@ -1929,7 +1929,7 @@ queries:
         Edit or create a file in the `/etc/audit/rules.d/` directory ending in `.rules` and add the following line:
 
         ```
-        -w <Path to sudo logfile> -p wa -k actions
+        -w <Path to sudo log file> -p wa -k actions
```
Comment (chris-rock) [inline core/mondoo-linux-security.mql.yaml:1932]: This looks like this is two args

**Review by chris-rock — APPROVED** [review]
Great addition @tas50

File: core/mondoo-linux-security.mql.yaml (hunk 1 above)
Comment (tas50) [reply core/mondoo-linux-security.mql.yaml:1932]: Everything in < > is a single argument. I think if folks are confused here they were probably confused before

**Review by misterpantz — APPROVED** [review]
Yay!

**Pull request merged**